release_docs/RELEASE.txt

```diff
@@ -70,8 +70,8 @@ New Features
 
     Fortran Library:
     ----------------
-    -
-
+    - Added Fortran H5Dfill_f, which is fully equivalent to the C API. It accepts pointers,
+      fill value datatype and datatype of dataspace elements, HDFFV-10734.
```

Comment:
I don't mean to pick on you :-), just simply need to know, do we still follow the pattern the document team required before?